application/libraries/Api/Command/V1/UserList.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace LimeSurvey\Api\Command\V1;
+
+use User;
+use Permission;
+use LimeSurvey\Api\Command\V1\Transformer\Output\TransformerOutputSurveyOwner;
+use LimeSurvey\Api\Command\{
+    CommandInterface,
+    Request\Request,
+    Response\Response,
+    Response\ResponseFactory
+};
+use LimeSurvey\Api\Auth\AuthSession;
+use LimeSurvey\Api\Command\Mixin\Auth\AuthPermissionTrait;
+
+class UserList implements CommandInterface
+{
+    use AuthPermissionTrait;
+
+    protected User $user;
+    protected AuthSession $authSession;
+    protected TransformerOutputSurveyOwner $transformerOutputSurveyOwner;
+    protected ResponseFactory $responseFactory;
+
+    /**
+     * Constructor
+     *
+     * @param User $user
+     * @param AuthSession $authSession
+     * @param TransformerOutputSurveyOwner $transformerOutputSurveyOwner
+     * @param ResponseFactory $responseFactory
+     */
+    public function __construct(
+        User $user,
+        AuthSession $authSession,
+        TransformerOutputSurveyOwner $transformerOutputSurveyOwner,
+        ResponseFactory $responseFactory
+    ) {
+        $this->user = $user;
+        $this->authSession = $authSession;
+        $this->transformerOutputSurveyOwner = $transformerOutputSurveyOwner;
+        $this->responseFactory = $responseFactory;
+    }
+
+    /**
+     * Run user list command
+     *
+     * @param Request $request
+     * @return Response
+     */
+    public function run(Request $request)
+    {
+        $sessionKey = (string) $request->getData('sessionKey');
+
+        if (
+            !$this->authSession
+                ->checkKey($sessionKey)
+        ) {
+            return $this->responseFactory
+                ->makeErrorUnauthorised();
+        }
+
+        if (!Permission::model()->hasGlobalPermission('users', 'read')) {
```

Comment:
The Permission class should be injected via the constructor to allow mocking in unit tests.

FYI, if you declare Permission as a constructor argument Permission::model() will be injected. You can see the logic defined for this here: application/libraries/DI.php